README.md

```diff
@@ -33,7 +33,13 @@ const algoliaIndexTransform = new AlgoliaIndexTransform({
   // The API Key for the new index
   destinationApiKey: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
   // The new index name
-  destinationIndexName: 'a new index'
+  destinationIndexName: 'a new index',
+  // Optional Limit the amount of records to transfer
+  limit: 20000,
```

Comment:
This stop the browsing after the first N elements have been passed, right?

Reply:
It will stop downloading records after `n` records have been recovered (kind of)